samples/go/models/requests.go

```diff
@@ -0,0 +1,99 @@
+package models
```

Comment:
Another streaming-related feedback, [SendTaskStreamingRequest](https://github.com/google/A2A/blob/main/specification/json/a2a.json#L1343) is missing.